README.md

```diff
@@ -1,16 +1,16 @@
-# Odin project sign up form
+# Odin dashboard project
 
-> One paragraph statement about the project.
+> The project contains some information .
 
 ![screenshot](./app_screenshot.png)
 
-Additional description of the project and its features.
+The project is one of the project that tells about the doer .
```

Comment:
I don't simply understand what you mean here, try to make every action you take be related to the project